examples/src/vsock/client.rs

```diff
@@ -0,0 +1,47 @@
+#![cfg_attr(not(unix), allow(unused_imports))]
+
+pub mod hello_world {
+    tonic::include_proto!("helloworld");
+}
+
+use hello_world::{greeter_client::GreeterClient, HelloRequest};
+use tokio_vsock::VsockStream;
+use tonic::transport::{Endpoint, Uri};
+use tower::service_fn;
+
+// Use vsock-loopback so we don't need to spin up a VM. Must match server.
+static TEST_CID: u32 = vsock::VMADDR_CID_LOCAL;
+// Arbitrarily choosen. Must match server.
+static TEST_PORT: u32 = 8000;
```

Comment:
Personally I would prefer to store this in a public variable and import them in both client & server.

However, I decided to repeat them in each place they are used, to match the other examples & tests.